src/unit/test_util.c

```diff
@@ -286,7 +286,9 @@ static int cache_exist(int fd) {
 int test_reclaimFilePageCache(int argc, char **argv, int flags) {
     UNUSED(argc);
     UNUSED(argv);
-    UNUSED(flags);
+
+    /* The test is incompatible with valgrind, skip it. */
```

Comment:
I'm just curious why it's incompatible. If you know, feel free to mention it in this code comment.

Reply:
i dont know the real reason, i am guessing valgrind may do some trick magic like mock the mmap to track the memory (and it did not handle this check), i am not an expert on it, just a guessing.